receiver/receivercreator/README.md

```diff
@@ -56,6 +56,34 @@ config:
    endpoint: `endpoint`:8080
 ```
 
+**receivers.&lt;receiver_type/id&gt;.resource_attributes**
+
+This setting controls what resource attributes are set on metrics emitted from the created receiver. These attributes can be set from [values in the endpoint](#rule-expressions) that was matched by the `rule`. These attributes vary based on the endpoint type. These defaults can be disabled by setting the attribute to be removed to an empty value. Note that the values can be dynamic and processed the same as in `config`.
+
+Note that the backticks below are not typos--they indicate the value is set dynamically.
+
+`type.pod`
+
+| Resource Attribute | Default       |
+|--------------------|---------------|
+| k8s.pod.name       | \`name\`      |
+| k8s.pod.uid        | \`uid\`       |
+| k8s.namespace.name | \`namespace\` |
```

Comment:
These use the same variables as rules. So in a rule you would say `name == "pod-name"` just as here name is the variable. For ports you have `pod.name` because container also has a name so the pod variables are namespaced to pod.

Reply:
Actually I think I misunderstood. You're saying you can add `pod.name` here to make it consistent with `pod.name` in port. Does that mean that for port type (to be podport) it should be `podport.name` instead of `name`? Not sure I like that but I also understand why it'd be nice for `pod.name` to be the same in both.